graylog2-web-interface/src/components/common/ExpandableList.tsx

```diff
@@ -29,16 +30,17 @@ type Props = {
  * of categories. Inside the categories the user has the possibility of doing a selection.
  * The ExpandableList can be used nested.
  */
-const ExpandableList = ({ children }: Props) => {
+const ExpandableList = ({ children, className }: Props) => {
   return (
-    <ul className={style.list}>
+    <ul className={`${style.list} ${className}`}>
```

Comment:
For the default props, this would result in `undefined` being part of the classes list. Instead we could do it like this:
```suggestion
    <ul className={className ? `${style.list} ${className}` : style.list}>
```